external-crates/move/crates/move-compiler/tests/move_2024/to_bytecode/clever_assertions.move

```diff
@@ -0,0 +1,25 @@
+module 0x42::m {
+    const Bar: u8 = 42;
+
+    #[error]
+    const X: vector<u8> = b"foo";
+
+    #[error]
+    const Foo: vector<u8> = b"Foo";
```

Comment:
We should have some wacky constant values, some nested vectors and what not

Reply:
Will do! That and the lovely test of

```
#[error]
const Lol: bool = false;

fun f() {
   assert!(Lol, Lol);
}
```